### PR TITLE
UI polish improve error messages and keyboard controls in user login and create form

### PR DESCRIPTION
**Pull request by RainingComputers**

### CODE REVIEW

File: meiki_ui/src/components/Entry.svelte [hunk 1]
```diff
@@ -1,23 +1,33 @@
 <script lang="ts">
     export let label: string
     export let password: boolean = false
+    export let onEnter: any = undefined
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/Entry.svelte:4]: Thank you!

File: meiki_ui/src/components/app/userForm/CreateUserForm.svelte [hunk 2]
```diff
@@ -20,7 +25,7 @@
             }
 
             error =
-                "An error has occured while creating the account, please try again later"
+                "An error has occured while creating the account, unable to connet to server"
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/app/userForm/CreateUserForm.svelte]: typo

Comment (alexsanjoseph) [inline meiki_ui/src/components/app/userForm/CreateUserForm.svelte]:
```suggestion
                "An error has occurred while creating the account, unable to connect to server"
```

File: meiki_ui/src/components/app/userForm/LoginUserForm.svelte [hunk 3]
```diff
@@ -20,7 +20,7 @@
             }
 
             error =
-                "An error has occured while logging in, please try again later"
+                "An error has occured while logging in, unable to connect to server"
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/app/userForm/LoginUserForm.svelte]:
```suggestion
                "An error has occurred while logging in, unable to connect to server"
```

**Review by alexsanjoseph — APPROVED** [review]

File: meiki_ui/src/components/app/userForm/CreateUserForm.svelte (hunk 2 above)
Comment (RainingComputers) [reply meiki_ui/src/components/app/userForm/CreateUserForm.svelte]: yep

Comment (RainingComputers) [reply meiki_ui/src/components/app/userForm/CreateUserForm.svelte]: yep

File: meiki_ui/src/components/app/userForm/LoginUserForm.svelte (hunk 3 above)
Comment (RainingComputers) [reply meiki_ui/src/components/app/userForm/LoginUserForm.svelte]: yep

**Pull request merged**